test/helpers/NonMerklizedOnchainIdentityDeployHelper.ts

```diff
@@ -0,0 +1,116 @@
+import { ethers, upgrades } from 'hardhat';
+import { Contract } from 'ethers';
+import { SignerWithAddress } from '@nomiclabs/hardhat-ethers/signers';
+
+export class NonMerklizedOnchainIdentityDeployHelper {
+  constructor(
+    private signers: SignerWithAddress[],
+    private readonly enableLogging: boolean = false
+  ) {}
+
+  static async initialize(
+    signers: SignerWithAddress[] | null = null,
+    enableLogging = false
+  ): Promise<NonMerklizedOnchainIdentityDeployHelper> {
+    let sgrs;
+    if (signers === null) {
+      sgrs = await ethers.getSigners();
+    } else {
+      sgrs = signers;
+    }
+    return new NonMerklizedOnchainIdentityDeployHelper(sgrs, enableLogging);
+  }
+
+  async deployIdentity(
```

Comment:
rename function to deployBalanceCredentialIssuer, and helper/script files accordingly too.